client/src/components/manageProjects/createNewEvent.js

```diff
@@ -116,7 +116,7 @@ const CreateNewEvent = ({
             className="create-form-button"
             onClick={() => {
               if (formValues.eventType == null) {
-                alert("Please select an event type.");
+                setFormErrors("Event type is blank. Select an event type.")
```

Comment:
This should be setting errors to an object with the key being the field name and the value being the error message:

`setFormErrors({ eventType: "Event type is blank. Select an event type." })`

Reply:
This is still not fixed.